### DISCUSSION TITLE
To Arabic

### DISCUSSION THREAD

**Discussion by ZAAFHachemrachid** [phil-opp/blog_os#1308] [Translations]
- Hello every one, I want first to thx the creator and contributors for this art. 
so i have Q :
1 . can i translate to Arabic ? .
2 .  if (yes) what level should i use ? (for your knowledge some work can't be translate but i well ask my professor  to help me in it)  .
3 . And if (yes) can do you consider using [astro Starlight](https://starlight.astro.build) for Doc .

**Comment by phil-opp** [comment]
Hi!


> * 1 . can i translate to Arabic ? .

Sure, an Arabic translation would be appreciated!

> * 2 .  if (yes) what level should i use ? (for your knowledge some work can't be translate but i well ask my professor  to help me in it)

I'm not sure what you mean with "level"?

> * . And if (yes) can do you consider using [astro Starlight](https://starlight.astro.build) for Doc .

I'm quite happy with our current static site generator https://www.getzola.org/ , so I don't have plans to switch to something else.
